### DISCUSSION TITLE
Requests to Telegram (sendMessage) fails: Connection timed out after 10001 milliseconds

### DISCUSSION THREAD

**Discussion by MatteoGheza** [def-studio/telegraph#487] [Q&A]
Inside `WebhookController`, inside a command function I called `$this->reply("text...");`.
I was reading Laravel logs, and I found this:
```
cURL error 28: Connection timed out after 10001 milliseconds (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for https://api.telegram.org/botTOKEN/sendMessage
```

When developing a similar system (with other framework and other Telegram bot library) I bump into the same error.
Is this a Telegram problem? Or is it related to the hosting?
What do you think I can do to troubleshoot this? Thanks.

**Comment by fabio-ivona** [comment]
Hi, can't reproduce the issue, could you please set up a sample repository showing this bug in action?

**Comment by MatteoGheza** [comment]
The problem is that I cannot reproduce the bug except on the prod env...
Maybe it's a wrongly configured hosting? It's from a very famous national hosting provider...

**Reply by rmkrylov** [reply]
I have a similar error, did you resolve the problem?

**Reply by fabio-ivona** [reply]
Cannot fix without a reproducing it, sorry
